Dashboard/va.gov.artemis.commands/Dsio/Observation/DsioSaveObservationToDifferentNamespaceCommand.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Generic;
+using VA.Gov.Artemis.Vista.Broker;
+using VA.Gov.Artemis.Vista.Utility;
+
+namespace VA.Gov.Artemis.Commands.Dsio.Observation
+{
+    /// <summary>
+    /// A command used to save an observation to VistA using an RPC  
+    /// </summary>
+    public class DsioSaveObservationToDifferentNamespaceCommand : DsioCommand
+    {
+        /// <summary>
+        /// Creates the command
+        /// </summary>
+        /// <param name="newBroker">A broker which allows communication with VistA and implements the IRpcBroker interface</param>
+        public DsioSaveObservationToDifferentNamespaceCommand(IRpcBroker newBroker) : base(newBroker) { }
+
+        /// <summary>
+        /// The IEN of the observation
+        /// </summary>
+        public string Ien { get; set; }
+
+        /// <summary>
+        /// The name of the RPC 
+        /// </summary>
+        public override string RpcName
+        {
+            get { return "WVRPCOR SAVEDATA"; }
+        }
+
+        /// <summary>
+        /// Add command arguments that are passed into the RPC 
+        /// </summary>
+        /// <param name="patientDfn">Patient DFN</param>
+        /// <param name="isLactating">Boolean shoing if the patient is lactating</param>
+        public void AddCommandArguments(string patientDfn, bool isLactating)
```

Comment:
is this only updating Lactation status?  Why not one AddCommandArguments for the entire SAVEDATA routine?

Reply:
Because you can save the Lactation without modifying the pregnancy. Lactation class does not depend on the pregnancy class, and vice-vesa.

Reply:
Ok, I see that.  You separated the functions because we may update one instead of the other and this way the caller doesn't need to have intrinsic knowledge how the RPC works.  I'm good with that.